pkg/claimhelper/claimhelper.go

```diff
@@ -0,0 +1,171 @@
+	if err != nil {
+		log.Fatalf("error unmarshalling configurations: %v", err)
+	}
+}
+
+func UnmarshalClaim(claimFile []byte, claimRoot *claim.Root) {
+	err := j.Unmarshal(claimFile, &claimRoot)
+	if err != nil {
+		log.Fatalf("error unmarshalling claim file: %v", err)
+	}
+}
+
+// writeClaimOutput writes the output payload to the claim file.  In the event of an error, this method fatally fails.
+func ReadClaimFile(claimFileName string) (data []byte, err error) {
+	data, err = os.ReadFile(claimFileName)
+	if err != nil {
+		log.Errorf("ReadFile failed with err: %s", err)
+	}
+	path, err := os.Getwd()
+	if err != nil {
+		log.Errorf("Getwd failed with err: %s", err)
+	}
+	log.Infof("Reading claim file at path: %s", path)
+	return data, nil
+}
+
+func GetConfigurationFromClaimFile(claimFileName string) (env *provider.TestEnvironment, err error) {
+	data, err := ReadClaimFile(claimFileName)
+	if err != nil {
+		log.Errorf("ReadClaimFile failed with err: %s", err)
+		return env, err
+	}
+	var aRoot claim.Root
+	fmt.Printf("%s", data)
+	UnmarshalClaim(data, &aRoot)
+	configJSON, err := j.Marshal(aRoot.Claim.Configurations)
+	if err != nil {
+		return nil, fmt.Errorf("cannot convert config to json")
+	}
+	err = j.Unmarshal(configJSON, &env)
+	return env, err
+}
+
+// marshalClaimOutput is a helper function to serialize a claim as JSON for output.  In the event of an error, this
+// method fatally fails.
+func MarshalClaimOutput(claimRoot *claim.Root) []byte {
+	payload, err := j.MarshalIndent(claimRoot, "", "  ")
+	if err != nil {
+		log.Fatalf("Failed to generate the claim: %v", err)
+	}
+	return payload
+}
+
+// writeClaimOutput writes the output payload to the claim file.  In the event of an error, this method fatally fails.
+func WriteClaimOutput(claimOutputFile string, payload []byte) {
+	err := os.WriteFile(claimOutputFile, payload, claimFilePermissions)
+	if err != nil {
+		log.Fatalf("Error writing claim data:\n%s", string(payload))
+	}
+}
+
+//no-lint:commentedOutCode
+func GenerateNodes() map[string]interface{} {
+	const (
+		nodeSummaryField = "nodeSummary"
+		cniPluginsField  = "cniPlugins"
+		nodesHwInfo      = "nodesHwInfo"
+		csiDriverInfo    = "csiDriver"
+	)
+	nodes := map[string]interface{}{}
+	nodes[nodeSummaryField] = diagnostics.GetNodeJSON()  // add node summary
+	nodes[cniPluginsField] = diagnostics.GetCniPlugins() // add cni plugins
+	nodes[nodesHwInfo] = diagnostics.GetHwInfoAllNodes() // add nodes hardware information
+	nodes[csiDriverInfo] = diagnostics.GetCsiDriver()    // add csi drivers info
+	return nodes
+}
+
+// createClaimRoot creates the claim based on the model created in
+// https://github.com/test-network-function/cnf-certification-test-claim.
+func CreateClaimRoot() *claim.Root {
+	// Initialize the claim with the start time.
+	startTime := time.Now()
+	c := &claim.Claim{
+		Metadata: &claim.Metadata{
+			StartTime: startTime.UTC().Format(DateTimeFormatDirective),
+		},
+	}
+	return &claim.Root{
+		Claim: c,
+	}
```

Comment:
```suggestion
	return &claim.Root{
		Claim: &claim.Claim{
		Metadata: &claim.Metadata{
			StartTime: startTime.UTC().Format(DateTimeFormatDirective),
		},
	},
	}
```

Reply:
same here, no change in code. Will try proposal

Reply:
Fixed